ramani-maplibre/src/main/java/org/ramani/compose/MapLibre.kt

```diff
@@ -226,7 +230,8 @@ private fun MapLibreMap.setupLocation(
     locationRequestProperties: LocationRequestProperties?,
     locationStyling: LocationStyling,
     userLocation: MutableState<Location>?,
-    renderMode: Int
+    renderMode: Int,
+    cameraMode: Int
```

Comment:
Nitpick again :see_no_evil: 
```suggestion
    cameraMode: Int,
```